utils/src/main/java/com/cloud/utils/net/NetUtils.java

```diff
@@ -1523,7 +1524,15 @@ public static boolean isIpWithtInCidrRange(final String ipAddress, final String
 
         final boolean isInRange = subnetUtils.getInfo().isInRange(ipAddress);
 
-        return isInRange;
+        return !isInRange;
```

Comment:
@DaanHoogland Nothing changes functionality wise. I reversed it so as it makes sense with the name of the function isNetworkorBroadCastIP() should return true if its a network or broadcast IP.